### PR TITLE
Release/0.7.1 to master branch

### PR DESCRIPTION
**Pull request by minsulee2**
The first regular release begins!

- Allow undefined gas_price
- Add _collectFee, _distributeFee
- Support data protocol version
- Add service app gas amount
- Add REST function call timeout
- Support consensus protocol version
- Change balance to PredefinedDbPaths.BALANCE
- Atomic function trigger (Exec results + isFailedTx())
- Update README.md
- Fix minor bugs

### CODE REVIEW

File: client/protocol_versions.json [hunk 1]
```diff
@@ -25,5 +25,8 @@
   },
   "0.7.0": {
     "min": "0.7.0"
+  },
+  "0.7.1": {
```
Comment (platfowner) [inline client/protocol_versions.json:29]: Could you make a separate PR for this change (to develop branch)?
I think it would be safer to have all changes reviewed as separate PRs to `develop` branch and the `develop` to `master` branch PR to be just an aggregation of such reviewed PRs?

Comment (platfowner) [reply client/protocol_versions.json:29]: Sorry, I missed this PR:
https://github.com/ainblockchain/ain-blockchain/pull/381

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: client/protocol_versions.json (hunk 1 above)
Comment (minsulee2) [reply client/protocol_versions.json:29]: Got this!

**Comment by minsulee2** [comment]
Thanks for the quick review! I am merging it now :)
